creation/lib/cgWDictFile.py

```diff
@@ -1,16 +1,44 @@
 # SPDX-FileCopyrightText: 2009 Fermi Research Alliance, LLC
 # SPDX-License-Identifier: Apache-2.0
 
-#
-# Project:
-#   glideinWMS
-#
-# File Version:
-#
 # Description:
 #   Glidein creation module Classes and functions needed to
 #   handle dictionary files
-#
+
+"""Dictionary files used in the Factory
+
+Main dictionaries are in the global configuration.
+Entry dictionaries are in the entry configuration.
+Common dictionaries are in both.
+The versioned dictionary are stored in timestamp versioned files
+
+Common dictionaries:
+attrs (cWDictFile.ReprDictFile, cgWConsts.ATTRS_FILE): attributes in the "entry" XML tag and "attrs" set as constants
+description (cWDictFile.DescriptionDictFile, cWConsts.DESCRIPTION_FILE, versioned): factory/entry description
+consts (cWDictFile.StrDictFile, cWConsts.CONSTS_FILE, versioned): constant attrs - TODO: not written?
+params (cWDictFile.ReprDictFile, cgWConsts.PARAMS_FILE): attrs that are not constant (const="False", default)
+submit (cWDictFile.ReprDictFile, cgWConsts.SUBMIT_ATTRS_FILE): submit configuration attributes
+vars (cWDictFile.VarsDictFile, cWConsts.VARS_FILE, versioned): condor variables
+untar_cfg (cWDictFile.StrDictFile, cWConsts.UNTAR_CFG_FILE, versioned): instructions for files to un-compress
+file_list (cWDictFile.FileDictFile, cWConsts.FILE_LISTFILE, versioned): list of files to transfers
+signature (cWDictFile.SHA1DictFile, cWConsts.SIGNATURE_FILE, versioned): files signatures for tampering verification
+
+Main dictionaries (common and):
+summary_signature (cWDictFile.SummarySHA1DictFile, cWConsts.SUMMARY_SIGNATURE_FILE):
+glidein (cWDictFile.StrDictFile, cgWConsts.GLIDEIN_FILE):
+build_cvmfsexec (cWDictFile.ReprDictFile, cgWConsts.CVMFSEXEC_BUILD_FILE): cvmfsexec releases
+frontend_descript (cWDictFile.ReprDictFile, cgWConsts.FRONTEND_DESCRIPT_FILE): frontend description
+gridmap (cWDictFile.GridMapDict, cWConsts.GRIDMAP_FILE, versioned):
+at_file_list (cWDictFile.FileDictFile, cgWConsts.AT_FILE_LISTFILE, versioned): list of files w/ intermediate priority (close to entry)
+after_file_list (cWDictFile.FileDictFile, cgWConsts.AFTER_FILE_LISTFILE, versioned): list of files loaded later (after entry)
+
+Entry dictionaries (common and):
+job_descript (cWDictFile.StrDictFile, cgWConsts.JOB_DESCRIPT_FILE)
+infosys (InfoSysDictFile, cgWConsts.INFOSYS_FILE)
+mongroup (MonitorGroupDictFile, cgWConsts.MONITOR_CONFIG_FILE)
+
+
+"""
```

Comment:
Nothing to be addressed. Just wanted to say that it is great to see documentation being added regarding the dictionary files the Factory uses! :clap: